storage/innobase/xtrabackup/test/suites/keyring/innodb_encryption_export.sh

```diff
@@ -66,7 +66,7 @@ ${XB_BIN} --prepare --apply-log-only --incremental-dir=$topdir/inc1 \
 	  --keyring-file-data=$keyring_file_plugin \
 	  --xtrabackup-plugin-dir=${plugin_dir} ${keyring_args}
 
-${XB_BIN} --prepare --apply-log-only --incremental-dir=$topdir/inc2 \
+${XB_BIN} --prepare --incremental-dir=$topdir/inc2 \
```

Comment:
is it possible to combine these two steps? (Just asking. Not asking you to change yet :))